collectors/python.d.plugin/alarms/README.md

```diff
@@ -0,0 +1,46 @@
+<!--
+title: "Alarms"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/alarms/README.md
+-->
+
+# Alarms - graphing active Netdata alarms over time
+
+This collector creates an 'Alarms' section with one line plot showing alarms by status. Alarm states are mapped to integer values according to the below mapping.
+
+```
+{'UNDEFINED': 0, 'CLEAR': 0, 'WARNING': 1, 'CRITICAL': 2}
+```
+
+## Charts
+
+Below is an example of the chart produced when running `stress-ng --all 2` for a few minutes. You can see the various warning and critical alarms raised. 
+
+![alt text](https://github.com/andrewm4894/random/blob/master/images/netdata/netdata-alarms-collector.jpg)
```

Comment:
This is an old dashboard screenshot (legend on the right), i think it is better to use current.

Reply:
will do